apps/src/javalab/JavalabEditor.jsx

```diff
@@ -50,6 +50,33 @@ const Dialog = makeEnum(
 );
 const DEFAULT_FILE_NAME = '.java';
 
+// Custom theme overrides
+const editorDarkModeThemeOverride = EditorView.theme(
+  {
+    '&': {
```

Comment:
mind adding a comment as to what each of these rules applies to?

Reply:
Yep done!